addon/components/bs4/bs-accordion/item.js

```diff
@@ -6,8 +6,8 @@ export default AccordionItem.extend({
   /**
    * @property classTypePrefix
    * @type String
-   * @default 'card'
+   * @default 'bg'
```

Comment:
This reminds me that we made a decision to move all docs to the `base` version, even for BS4-specific stuff.